apps/src/StudioApp.js

```diff
@@ -1579,8 +1586,8 @@ StudioApp.prototype.handleHideSource_ = function (options) {
       $('body').append(wireframeSendToPhone);
     }
     document.body.style.backgroundColor = '#202B34';
-  // For share page on mobile, do not show this part.
-  } else if (!options.embed && !(this.share && dom.isMobile())) {
+  // For share page, do not show this part.
+  } else if (!options.embed && !this.share) {
```

Comment:
Can we add a TODO here (and perhaps a pivotal item) to explore whether this block is now dead code that can be deleted after hoc?